app/src/main/kotlin/com/wire/android/ui/home/conversationslist/ConversationListViewModel.kt

```diff
@@ -143,7 +144,10 @@ class ConversationListViewModel @Inject constructor(
                 val otherUser = details.otherUser
                 GeneralConversation(
                     ConversationType.PrivateConversation(
-                        userInfo = UserInfo("", UserStatus.NONE), //TODO Get actual status and avatar
+                        userInfo = UserInfo(
+                            otherUser.completePicture?.let { UserAvatarAsset(it) },
```

Comment:
So we load here in the conversation avatar row the complete picture but the preview one in the HomeTopBar?

Reply:
Mistakes were made 😬

I'll use the preview for these tiny guys.